scripts/hid_configurator/requirements.txt

```diff
@@ -1,2 +1,8 @@
+docutils
+pygments
+pypiwin32
+kivy_deps.sdl2==0.1.22
```

Comment:
Check if always necessary to install these dependencies

Reply:
Platform information added